### DISCUSSION TITLE
Slow rendering when filters 'react' to one another

### DISCUSSION THREAD

**Discussion by john-conroy** [appbaseio/reactivesearch#1912] [Q&A]
A project I'm working on is considering migrating from searchkit. We've started the process of integrating reactivesearch, but are running into rendering performance issues. We want to replicate searchkit's behavior for filter aggs where each filter has an 'and' relationship to every other sensor component but itself. 

We have 12 filters and when updating it takes ~10-50ms to receive the response from elasticsearch, but ~5-10 seconds to render the updated results and filter aggs.

Here's a basic [reproducer](https://codesandbox.io/s/wispy-tree-6i0opi?file=/src/index.js) forked from one of your examples.

Thanks!

**Comment by siddharthlatest** [comment]
@john-conroy Thanks for bringing up this issue. 

What I see in the demo is that the time reported from ES in the UI is different from the net network time. What I see is that as soon as the network requests are resolved, the UI updates (perceived update time feels sub second). Can you confirm this? Keep the network tab open to see when the request is actually resolved.
